### ISSUE TITLE
Recent Change to Favoriting GIFs

### ISSUE THREAD

**Issue by Aezaeo** [Dastan21/BDAddons#83]
I see the most recent update took away the ability to favorite GIFs, any particular reason for this? I understand that Discord has it built in but I had already reached the cap for favoriting via the built in favorites so I was using FavoriteMedia to favorite excess gifs. It was very useful for myself. 

I'm curious if this feature could be added back or if it's gone for good.  It was extremely beneficial to my specific use case.

**Comment by Dastan21** [comment]
Well, I could add a new custom tab for GIFs. Instead of replacing the existing one (and make it toggleable in plugin settings).

**Comment by Aezaeo** [comment]
That would be cool actually, It would eliminate the "gifs" category I had created for saving gifs that got placed within the images tab. Also if memory serves me correctly, gifs that were placed in the category I created wouldn't actually play? Some would have the broken image link icon instead of playing through. I cant remember if those were "gifv" or "gif" format.

Either way, a dedicated GIFs tab would be useful and cool.
